TileFormats/Instanced3DModel/README.md

```diff
@@ -20,25 +20,39 @@ Instanced 3D Model maps well to the [ANGLE_instanced_arrays](https://www.khronos
 
 A tile is composed of a header section immediately followed by a body section.
 
+
 **Figure 1**: Instanced 3D Model layout (dashes indicate optional fields).
 
-![](figures/layout.png)
+![header layout](figures/header-layout.png)
```

Comment:
In the Cesium implementation `span` is called `scale`.  What is the standard terminology for quantization?  Let's use it in all cases.